### PR TITLE
Refactor 

### LINKED ISSUE #64: Add prop types for all relevant files

### LINKED ISSUE #65: Deploy repo via GitHub pages

### PR DESCRIPTION
**Pull request by nicolegooden**
#### What's this PR do?  

+ Removes white space from files
+ Changes style for activity cards displayed on the `BrowsePage` so that only the title shows up for each card game, but styles for all other activity types remain unchanged
+ Adds `prop types` for any component that is receiving props in App's render.  I checked that other components rendered outside of App are accounted for as well (none had props)
+ Removes `About` from top of Homepage
+ Removes functions from inside of functional components; they are now variables with the same purpose
+ Changes styles for the hover state of form's choices - on hover, the user will now see the choice pop out (increase in text size) and the text shadow has been reduced
+ Deploys this repo via GH pages

#### Where should the reviewer start?

These changes exist in most of the files, feel free to just view the files in order.  Most changes happen at the bottom of the file or in the file's `render()`. 

#### How should this be manually tested?  

Pull down this version of code and ensure all functionality is in tact (mostly referring to form here).  Hover over the choices in the `Form` component to see the differences in styling.  In the browser, ensure there are no errors for failed prop types.

#### Any background context you want to provide?  

N/A

#### What are the relevant tickets?  

Closes #64 
Closes #65

### CODE REVIEW

File: src/Components/Form/Form.js [hunk 1]
```diff
@@ -254,30 +220,34 @@ export class Form extends Component {
...
+Form.propTypes = {
+  route: PropTypes.object,
+  goBack: PropTypes.func.isRequired,
+  setHistory: PropTypes.func.isRequired,
+  resetState: PropTypes.func.isRequired,
+  activities: PropTypes.array.isRequired,
+  setActivities: PropTypes.func.isRequired,
+  updateActivityAnswers: PropTypes.func.isRequired,
+  determineRandomActivity: PropTypes.func.isRequired
+}
```
Comment (BlakeDonn) [inline src/Components/Form/Form.js:253]: These all look good to me!

**Review by BlakeDonn — APPROVED** [review]
Thanks for cleaning up all the whitespace and normalizing syntax styling. Looks a lot better overall

File: package.json [hunk 2]
```diff
@@ -2,10 +2,13 @@
   "name": "indecision-maker",
   "version": "0.1.0",
   "private": true,
+  "homepage": "http://nicolegooden.github.io/indecision-maker",
```
Comment (Atos20) [inline package.json:5]: Thanks for taking the time to deploy the repo.

File: src/Components/App/App.js [hunk 3]
```diff
@@ -3,16 +3,16 @@ import { Homepage } from '../HomePage/Homepage';
...
   getAllPodcasts, 
   getAllCardGames, 
   getAllMusic, 
   getAllBoardGames} 
-  from '../../apiCalls.js'
-import { BrowsePage } from '../BrowsePage/BrowsePage';
-import { Footer } from '../Footer/Footer';
-import { ResultPage } from '../ResultPage/ResultPage';
-import { DetailsPage } from '../DetailPage/DetailsPage';
+  from '../../apiCalls.js';
```
Comment (Atos20) [inline src/Components/App/App.js:15]: I like the way this is ordered now.

File: src/Components/BrowsePage/BrowsePage.js [hunk 4]
```diff
@@ -2,49 +2,50 @@ import React from 'react'
...
+    const activityInformation =
+        props.data.map((e, i) => {
             return (
                 <li key ={i}className="hexa">
                     <div className="inner-hexagon">
-                        <img className="hexa-image" 
-                        src={e.image_poster || e.image_100 || e.image} 
-                        alt={e.song_title || e.instructions || e.podcast_title || e.name}
-                        />
+                        {(e.image_poster || e.image_100 || e.image) && <img className="hexa-image"
```
Comment (Atos20) [inline src/Components/BrowsePage/BrowsePage.js:14]: Thanks for taking the time and handling this error.

File: src/Components/BrowsePage/BrowsePage.scss [hunk 5]
```diff
@@ -77,14 +77,20 @@ $hex-size: 6em; // Only change size here. Best used with sizes from 5em - 10em.
...
                     color: $darkPurple;
-                    margin: -10%
+                    margin: -10%;
+                }
+
+                .title-copy {
+                  font-size: 1.3em;
+                  font-family: 'Comfortaa', cursive; 
+                  color: $darkPurple;
+                  margin-top: 50%;
```
Comment (Atos20) [inline src/Components/BrowsePage/BrowsePage.scss:93]: What a great idea.

**Review by Atos20 — APPROVED** [review]
Thanks for implementing `react-propTypes`

**Pull request merged**